integration/tests/epochs/base_suite.go

```diff
@@ -117,8 +118,15 @@ func (s *BaseSuite) SetupTest() {
 	go lib.LogStatusPeriodically(s.T(), s.Ctx, s.log, s.Client, 5*time.Second)
 }
 
+func (s *BaseSuite) TearDownTest() {
+	s.log.Info().Msg("================> Start TearDownTest")
```

Comment:
still need those logs?

Reply:
Yes these help with debugging we have them on all the epoch related integration tests.